cirq-core/cirq/contrib/paulistring/convert_gate_set.py

```diff
@@ -12,22 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cirq import circuits, transformers
+from cirq import circuits, transformers, _compat
 
-from cirq.contrib.paulistring.convert_to_pauli_string_phasors import ConvertToPauliStringPhasors
+from cirq.contrib.paulistring.clifford_target_gateset import CliffordTargetGateset
 
 
+@_compat.deprecated(
```

Comment:
isn't this too late for a 0.16 deprecation

Reply:
As discussed during the design sync; there's no time for a deprecation now so directly deleted the deprecated objects.